src/libraries/ProtocolFeeLibrary.sol

```diff
@@ -2,11 +2,11 @@
 pragma solidity ^0.8.20;
 
 library ProtocolFeeLibrary {
-    // Max protocol fee is 25% (2500 bips)
-    uint16 public constant MAX_PROTOCOL_FEE = 2500;
+    // Max protocol fee is 0.1%
```

Comment:
could you also add comment here that its denominated in pips

Reply:
✅